### ISSUE TITLE
Vulkan: Broken graphics in Hatsune Miku: Project Diva F after #1828

### ISSUE THREAD

**Issue by Nekotekina** [RPCS3/rpcs3#1932]
This is not from the latest PR.
![bug00](https://cloud.githubusercontent.com/assets/6028184/16902040/316d5a94-4c5c-11e6-8f5d-f911d4d36803.png)
![bug0](https://cloud.githubusercontent.com/assets/6028184/16902044/3cbc4edc-4c5c-11e6-8c4a-1e76589fe4d5.png)

Weird color areas with blur edges randomly appears everywhere.

**Comment by Nekotekina** [comment]
Update: it's caused by commit 6ae54ae27b048ce260d9067b09d08bcbbfdf1a68

**Comment by Nekotekina** [comment]
![bug000](https://cloud.githubusercontent.com/assets/6028184/16902923/d7b467c0-4c76-11e6-84b1-b6c13f3426e4.png)

**Comment by kd-11** [comment]
@Nekotekina Which GPU are you using?

**Comment by Nekotekina** [comment]
@kd-11 Radeon HD 7500, why you ask, you can't reproduce the issue? Something similar was reported in another issue. I think maybe revert this commit.

**Comment by Kristiqn5** [comment]
same issue here on latest build

![2837c5f7616a5a4d398bd70b265b82a7](https://cloud.githubusercontent.com/assets/13494982/17059348/66e3f104-502d-11e6-909d-97b2c6cfcef1.gif)

I have GTX 750 Ti
